packages/@react-stately/tooltip/src/useTooltipTriggerState.ts

```diff
@@ -101,7 +101,7 @@ export function useTooltipTriggerState(props: TooltipTriggerProps = {}): Tooltip
         delete tooltips[id];
         globalCooldownTimeout = null;
         globalWarmedUp = false;
-      }, TOOLTIP_COOLDOWN);
+      }, closeDelay);
```

Comment:
This should probably be `Math.max(TOOLTIP_COOLDOWN, closeDelay)`
I don't think there is a case where we'd like for the global cooldown to be 0.

The two values happened to be the same for us in Spectrum land, and so they've been conflated a little, not your fault. Cooldown is not the same thing as close delay.

Close delay is how long a tooltip stays open after moving off the target. Cooldown is how long, after being warmed up, until they require a warm up again. Warm up meaning that all subsequent tooltips appear immediately.

Reply:
Done. Thanks for clarifying. I had a suspicion something like that was going on.

Reply:
@snowystinger I wonder if we should introduce a new prop for setting the cooldown as well. That behavior feels pretty spectrum-y (forget if we've discussed this before)

Reply:
Yeah, had that thought as well. I think it's not as important to change though and mostly unnoticeable. So I'm content to leave it without a separate prop until someone asks for it with a compelling usecase